prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-actions/adjudicator-actions.component.html

```diff
@@ -1,26 +1,28 @@
 <ng-container *ngIf="enrollee">
 
-  <button mat-flat-button
-          color="primary"
-          class="mb-5 d-block"
-          [disabled]="!enrollee"
-          [matMenuTriggerFor]="menu">
-    Adjudicator Actions <mat-icon>keyboard_arrow_down</mat-icon>
-  </button>
+  <div [class]="(mode === 'column') ? 'd-flex flex-column-reverse' : ''">
```

Comment:
I would probably have had `class="d-flex" [class]="(mode === 'row') ? 'flex-row' : 'flex-column-reverse'"` or something equivalent since I haven't tested the snippet. It keeps things explicit, avoids the empty string ternary, and the `d-block` on the button since you're using flexbox you shouldn't need it

Reply:
Changed to `[class]="(mode === 'row') ? 'flex-row' : 'd-flex flex-column-reverse'"`. Adding `d-flex` to the `flex-row` throws the Adjudicator Actions out of alignment.